supervisor/backups/manager.py

```diff
@@ -208,6 +215,8 @@ async def import_backup(self, tar_file: Path) -> Backup | None:
             backup.tarfile.rename(tar_origin)
 
         except OSError as err:
+            if err.errno == errno.EBADMSG:
+                self.sys_resolution.unhealthy = UnhealthyReason.OSERROR_BAD_MESSAGE
```

Comment:
Could be a mount

Reply:
Actually this one cannot be a mount. This is in `import_backup`. This is used when user uploads a backup from an external system via the API. Supervisor puts this backup directly in the backup folder, it will not put it into a mount.